### PR TITLE
feat(sw-review): add PR comment review utility skill

### PR DESCRIPTION
**Pull request by MacAttak**
## Summary

- New `sw-review` utility skill for PR comment review and response
- Fetches all 3 comment types: issue comments (REST), review comments (REST), thread resolution (GraphQL)
- Groups comments by priority: unresolved threads > new comments > resolved threads
- Enables inline response via `gh api` with thread resolution via GraphQL mutation
- Graceful degradation to URL display without `gh` CLI
- Part of workflow-commands work unit (unit 2 of 3)

## Acceptance Criteria

| # | Criterion | Status | Evidence |
|---|-----------|--------|----------|
| AC1 | SKILL.md follows skill anatomy template | PASS | 18 tests (frontmatter, sections, word count, tools) |
| AC2 | PR detection from current branch | PASS | 5 tests (branch detect, PR list, merged fallback, disambiguation, prTool) |
| AC3 | All three comment types fetched | PASS | 5 tests (issue REST, review REST, GraphQL threads, all distinct, cap) |
| AC4 | Comments grouped by status/priority | PASS | 5 tests (unresolved priority, grouping, author, timestamp, file/line) |
| AC5 | User can respond to comments | PASS | 4 tests (POST reply, resolveReviewThread, gh api, no gh pr edit) |
| AC6 | Graceful degradation without gh | PASS | 4 tests (URL construction, install message, no STOP, degradation) |
| AC7 | No workflow state mutation | PASS | 5 tests (no write, no lock, read-only, no Write tool, no TodoWrite) |
| AC8 | Documentation updated | PASS | DESIGN.md, CLAUDE.md, AGENTS.md, opencode adapter, build test |
| AC9 | Failure modes cover edge cases | PASS | 6 tests (detached HEAD, no PR, no comments, rate limit, bullet count) |

## Blast Radius

| Module | Touched | Failure scope |
|--------|---------|---------------|
| `core/skills/sw-review/SKILL.md` | New file | Local — no existing dependencies |
| `DESIGN.md` | Modified (skill table, directory, counts) | Local — documentation |
| `CLAUDE.md` | Modified (skill table) | Local — documentation |
| `AGENTS.md` | Modified (skill table) | Local — documentation |
| `adapters/opencode/commands/sw-review.md` | New file | Local — opencode adapter |
| `tests/test-sw-review-skill.sh` | New file | Local — test only |
| `tests/test-claude-code-build.sh` | Modified (count 21→22) | Local — test only |

## Gate Results

| Gate | Status | Findings (B/W/I) |
|------|--------|-------------------|
| Build | PASS | 0/0/0 |
| Tests | PASS | 0/0/0 |
| Security | PASS (with WARNs) | 0/2/5 |
| Wiring | PASS | 0/0/10 |
| Semantic | SKIP — markdown files only, no executable code |
| Spec | PASS | 0/0/0 (9/9 AC verified) |

2 security WARNs (API path parameter validation, GraphQL input sanitization) — spec-completeness items, not active vulnerabilities. Reviewer approved without blocking.

## Evidence

- 59 skill structure tests (`tests/test-sw-review-skill.sh`)
- 182 build validation tests (`tests/test-claude-code-build.sh`)
- 443 pytest evals, 42 hook tests — all passing

🤖 Generated with [Specwright](https://github.com/Obsidian-Owl/specwright)

### CODE REVIEW

File: core/skills/sw-review/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  preferred. Default to `gh` if unset.
+
+**Comment fetching (MEDIUM freedom):**
+- Fetch all three comment types via `gh api`:
+  - Issue comments (general PR conversation): `GET /repos/{owner}/{repo}/issues/{n}/comments`
+  - Review comments (inline code comments): `GET /repos/{owner}/{repo}/pulls/{n}/comments`
+  - Thread resolution state: GraphQL query on `reviewThreads` with `isResolved` field
+- Cap fetched comments at 50 per type using `--per-page 50`. If the response
+  indicates more exist, display a truncation notice so the user knows the list
+  is incomplete.
```
[the remark this answers is not in the document]
Comment (MacAttak) [reply core/skills/sw-review/SKILL.md]: Fixed in 1956ad4. Now uses 'if exactly 50 returned, assume more may exist' heuristic and notes that pagination state is in Link headers, not JSON body.

File: core/skills/sw-review/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  resolved threads. Do not interleave types.
+- Within each group, order by timestamp (newest first for unresolved, oldest
+  first for general comments).
+
+**Responding and resolving (MEDIUM freedom):**
+- To post a reply to a review comment, use `gh api` with an HTTP POST:
+  `gh api --method POST /repos/{owner}/{repo}/pulls/{n}/comments/{id}/replies`.
+- To post an issue comment reply, POST to
+  `/repos/{owner}/{repo}/issues/{n}/comments`.
+- To resolve a review thread, call the `resolveReviewThread` GraphQL mutation
```
[the remark this answers is not in the document]
Comment (MacAttak) [reply core/skills/sw-review/SKILL.md]: Fixed in 1956ad4. Clarified that issue comments are top-level with no threading: 'POST to .../issues/{n}/comments with a body field. Issue comments have no threading — there is no in_reply_to_id for this endpoint.'

File: core/skills/sw-review/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+**PR detection (MEDIUM freedom):**
+- Detect the current branch with `git branch --show-current`. If the result
+  is empty or HEAD is detached, report the detached HEAD condition and stop.
+- Discover the associated PR using `gh pr list --head {branch} --json number,title,url`.
+  If no open PR is found, retry with `--state merged` as a fallback for merged PRs.
+- If multiple PRs are returned, use AskUserQuestion to disambiguate — present
+  titles and numbers so the user can select one. Do not guess.
+- Read `config.git.prTool` before invoking `gh`; this determines which CLI is
+  preferred. Default to `gh` if unset.
```
[the remark this answers is not in the document]
Comment (MacAttak) [reply core/skills/sw-review/SKILL.md]: Fixed in 1956ad4. Now explicitly scoped: 'if the value is not gh or is unset, degrade to URL display. sw-review only supports the gh CLI.'
